brave_testrun_generator.py

```diff
@@ -654,290 +573,356 @@ def android_testruns(milestonever):
-                if("QA Pass-macOS" not in label_names and
-                        "feature/crypto-wallets" in label_names and
-                        "OS/Windows" not in label_names and
-                        "OS/Linux" not in label_names and
-                        "QA/No" not in label_names and
-                        "tests" not in label_names):
-                    mac_checklist.append(output_line)
+                iPhone_checklist.append(output_line)
 
-                if("QA Pass-Win64" not in label_names and
-                        "feature/crypto-wallets" in label_names and
-                        "OS/macOS" not in label_names and
-                        "OS/Linux" not in label_names and
-                        "QA/No" not in label_names and
-                        "tests" not in label_names):
-                    win64_checklist.append(output_line)
+            if("QA Pass - iPhone X" not in label_names and
+                    "iPad" not in label_names and
+                    "tests" not in label_names):
+                iPhoneX_checklist.append(output_line)
 
-                if("QA Pass-Linux" not in label_names and
-                        "feature/crypto-wallets" in label_names and
-                        "OS/Windows" not in label_names and
-                        "OS/macOS" not in label_names and
-                        "QA/No" not in label_names and
-                        "tests" not in label_names):
-                    linux_checklist.append(output_line)
+    print("Release Notes:")
+    print(release_notes)
+    for line in release_notes:
+        relline += line +"\n"
+        print(relline)
+    print("")
 
-    print("Mac Checklist:\n")
-    bigline = "## Per release specialty tests\n"
-    for line in mac_checklist:
+    print("iPad Checklist:")
+    bigline = "## Per release speciality tests\n"
+    for line in iPad_checklist:
+        bigline += line + "\n"
+    bigline = bigline + ios_template
+    print(bigline)
+    print("")
+    iPad_Title = "Manual test run for " + ios_key +\
+        " on iPad running iOS12"
+    iPad_List = ["ipad",
+                 "release-notes/exclude",
+                 "tests",
+                 "QA/Yes"]
+
+    if args.test is None:
+        ios_repo.create_issue(title=iPad_Title,
+                              body=bigline,
+                              assignees=["srirambv", "kjozwiak"],
+                              milestone=ios_milestone[ios_key],
+                              labels=iPad_List)
+
+    print("--------------------------------------------------------")
+
+    print("iPhone Checklist:")
+    bigline = "## Per release speciality tests\n"
+    for line in iPhone_checklist:
         bigline += line + "\n"
-    bigline = bigline + cryptowallet_template
+    bigline = bigline + ios_template
     print(bigline)
     print("")
-    macTitle = "Manual test run on OS X for " + milestonever
+    iPhone_Title = "Manual test run for " + ios_key +\
```

Comment:
Looks like we're hard-coding `iOS` to all titles when created passes for `iPhone`. We depreciated `iOS 12` a while ago.

Should try generating something a long the lines of the following:

![image](https://user-images.githubusercontent.com/2602313/121423165-f9703300-c93d-11eb-8f34-7ab65abc0d0c.png)

Lets discuss which device should be used for `iOS 13` so we can update the titles. We can always update the titles if lets say I decide to run through `iOS 13` on the `iPad`. But we should set a default.

Reply:
Same as above. Can remove the iOS version and can edit the title while running the manual pass for now